README.md

```diff
@@ -38,11 +38,23 @@ API KEY validation algorithm:
 6. lookup `TOKEN_HASH` in database; return `true` iff found
 
 ## Environment variables
-| NAME                  | DESCRIPTION                                                                    |
-| ----------------------| ------------------------------------------------------------------------------ |
-| DB_SSL_OPTION         | To turn TLS on this value must be set as _“ssl=true”_. Otherwise must be empty. |
-| TOKEN_DB_BCRYPT_SALT  | Salt used for the hashing algorithm (bcrypt) to hash tokens before being stored in DB. |
-| TOKEN_API_HMAC_SECRET | HMAC secret to create the signature for the API Key. |
+| NAME                    | DESCRIPTION                                                                    |
+| ----------------------- | ------------------------------------------------------------------------------ |
+| `ADMIN_PORT`            | The port number to listen for Dropwizard admin requests on. Defaults to `8081`. |
+| `CERTS_PATH`            | If set, add all certificates in this directory to the default Java truststore. |
+| `DB_HOST`               | The hostname of the database server. |
+| `DB_PASSWORD`           | The password for the `DB_USER` user. |
+| `DB_SSL_OPTION`         | To turn TLS on this value must be set as _“ssl=true”_. Otherwise must be empty. |
```

Comment:
Could you change the formatting of the example to match the others please?

Reply:
I can